apps/cookbook/src/app/showcase/styling-list/styling-list-showcase.html

```diff
@@ -0,0 +1,119 @@
+  <li>Lorem ipsum dolor sit amet</li>
+  <li>Donec ullamcorper, risus id venenatis</li>
+</ul>
+<h2>Unordered List /w Nested Lists Example</h2>
+<ul>
+  <li>Maecenas purus sapien</li>
+  <li>Pellentesque ultricies mi</li>
+  <li>
+    Dignissim et lorem mollis
+    <ul>
+      <li>Duis eget tincidunt sapien</li>
+      <li>Phasellus viverra ipsum sed</li>
+      <li>
+        Donec pulvinar aliquet turpis
+        <ul>
+          <li>Praesent vitae augue sed</li>
+          <li>Vitae convallis enim lobortis</li>
+          <li>Nisi condimentum ultricies nec</li>
+        </ul>
+      </li>
+    </ul>
+  </li>
+</ul>
+<h2>Ordered List Example</h2>
+<ul>
+  <li>Nulla cursus sem nulla</li>
+  <li>Sed egestas lorem pharetra</li>
+  <li>Pellentesque habitant morbi</li>
+</ul>
+<h2>Ordered List /w Nested Lists Example</h2>
+<ol>
+  <li>Proin consectetur pharetra</li>
+  <li>Aenean lobortis porta dictum</li>
+  <li>
+    Vestibulum in tortor vulputate
+    <ol>
+      <li>Fusce iaculis ante ac diam</li>
+      <li>Ac elementum nisi eleifend</li>
+      <li>
+        Maecenas nec convallis orci
+        <ol>
+          <li>Praesent ut felis quis</li>
+          <li>Etiam quis placerat neque</li>
+          <li>Nunc malesuada luctus varius.</li>
+        </ol>
+      </li>
+    </ol>
+  </li>
+</ol>
+<h2>Mixed List</h2>
+<ul>
+  <li>Proin at pharetra ligula</li>
+  <li>Integer id diam eleifend ac</li>
+  <li>
+    Sit amet blandit tortor dignissim
+    <ol>
+      <li>Consectetur adipiscing elit</li>
+      <li>Mauris fringilla sodales nulla</li>
+      <li>
+        In pulvinar massa ac porttitor
+        <ul>
+          <li>Proin at pharetra ligula</li>
+          <li>Suscipit erat a, aliquet nisi</li>
+          <li>Curabitur tempus dolor massa</li>
+        </ul>
+      </li>
+    </ol>
+  </li>
+</ul>
+<h2>Text Overflow in List</h2>
+<ul>
+  <li>
+    Proin at pharetra ligula, donec ullamcorper, risus id venenatis. Integer id diam eleifend ac,
+    consectetur adipiscing elit. Fusce iaculis ante ac diam ac elementum nisi eleifend in pulvinar
+    massa ac porttitor.
+  </li>
+</ul>
+<h2>How to "opt-out" list styling</h2>
+<p>
+  In some cases you may not want the default styling. A common example is a navigation menu where an
+  HTML list is used but styled to not look like a list. You’re probably already overriding the
+  default user agent styles for lists. In addition you may want to override some or all of the Kirby
+  default list styling. Which declarations should be made could differ depending on the specific
+  case.
+</p>
+
+<cookbook-code-viewer [html]="defaultListExample.template">
+  <cookbook-styling-list-example #defaultListExample></cookbook-styling-list-example>
+</cookbook-code-viewer>
+<br />
+<p>To remove the custom markers (bullets):</p>
+<cookbook-code-viewer [css]="defaultListExample.styles">
+  <cookbook-styling-list-example #defaultListExample></cookbook-styling-list-example>
+</cookbook-code-viewer>
```

Comment:
Replace `<cookbook-styling-list-example>` with embedded code (just like the next CSS code block)